WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsConnectedViewController.swift

```diff
@@ -66,20 +58,19 @@ final class CardReaderSettingsConnectedViewController: UIViewController, CardRea
 // MARK: - View Configuration
 //
 private extension CardReaderSettingsConnectedViewController {
-    func checkForCardReaderUpdate() {
-        guard let viewModel = viewModel else {
-            return
-        }
-
-        viewModel.checkForCardReaderUpdate()
-    }
-
     func onViewModelDidUpdate() {
         configureSections()
         configureTable()
         configureUpdateView()
     }
 
+    func isReaderUpdateAvailable() -> Bool {
+        guard let viewModel = viewModel else {
+            return false
+        }
+        return viewModel.readerUpdateAvailable == true
```

Comment:
```suggestion
        return viewModel?.readerUpdateAvailable == true
```

This could also be inlined at the call sites of this function, if desired...

Reply:
Opened https://github.com/woocommerce/woocommerce-ios/issues/5427